### ISSUE TITLE
[Bug]: In Qingping mode, CGPR1 not working, Sensors  light and motion always Unknown

### ISSUE THREAD

**Issue by EastWoodYang** [custom-components/ble_monitor#860]
### What happened?

In Qingping mode
Qingping CGPR1 not working, Sensors  light and motion always Unknown

![image](https://user-images.githubusercontent.com/9335046/167750035-eb633ce9-a102-45aa-b22d-b76ce0b14a98.png)

### Sensor type

CGPR1 by Qingping

**Comment by Ernst79** [comment]
I need more info to help you. Are the RSSI and illumination updating normally. Anything in the log?

**Comment by EastWoodYang** [comment]
Yes, RSSI and illumination updating normally, I will enable debug logging later.

**Comment by Ernst79** [comment]
if you don't see anything special in the logs, try to enable report_unknown for your MAC. It might help me to figure out why certain BLE advertisements aren't parsed.

**Comment by EastWoodYang** [comment]
```
2022-05-11 21:29:13 INFO (Thread-4) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 582d3460de98: 043e350d0113000098de60342d580100ff7fbe0000000000000000001b0201061716cdfd881298de60342d580201640f01830904d1000000
2022-05-11 21:29:13 DEBUG (MainThread) [custom_components.ble_monitor.binary_sensor] Data binary sensor received: {'rssi': -66, 'packet': 131, 'battery': 100, 'illuminance': 209, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}
2022-05-11 21:29:13 DEBUG (MainThread) [custom_components.ble_monitor.sensor] Data measuring sensor received: {'rssi': -66, 'packet': 131, 'battery': 100, 'illuminance': 209, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}
2022-05-11 21:29:13 DEBUG (MainThread) [custom_components.ble_monitor.device_tracker] Data device tracker received: {'is connected': True, 'mac': '582D3460DE98', 'rssi': -66, 'gateway_id': 'ble_monitor'}
2022-05-11 21:29:15 INFO (Thread-4) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 582d3460de98: 043e320d0113000098de60342d580100ff7fb4000000000000000000180201061416cdfdc81298de60342d58080400d100000f0186
2022-05-11 21:29:15 DEBUG (MainThread) [custom_components.ble_monitor.device_tracker] Data device tracker received: {'is connected': True, 'mac': '582D3460DE98', 'rssi': -76, 'gateway_id': 'ble_monitor'}
2022-05-11 21:29:15 INFO (Thread-4) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 582d3460de98: 043e320d0113000098de60342d580100ff7fb2000000000000000000180201061416cdfdc81298de60342d58080400d100000f0186
2022-05-11 21:29:15 DEBUG (MainThread) [custom_components.ble_monitor.device_tracker] Data device tracker received: {'is connected': True, 'mac': '582D3460DE98', 'rssi': -78, 'gateway_id': 'ble_monitor'}
2022-05-11 21:29:16 INFO (Thread-4) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 582d3460de98: 043e320d0113000098de60342d580100ff7fb2000000000000000000180201061416cdfdc81298de60342d58080400d100000f0186
2022-05-11 21:29:16 DEBUG (MainThread) [custom_components.ble_monitor.device_tracker] Data device tracker received: {'is connected': True, 'mac': '582D3460DE98', 'rssi': -78, 'gateway_id': 'ble_monitor'}
2022-05-11 21:29:17 INFO (Thread-4) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 582d3460de98: 043e320d0113000098de60342d580100ff7fc0000000000000000000180201061416cdfdc81298de60342d58080400d100000f0186
2022-05-11 21:29:17 DEBUG (MainThread) [custom_components.ble_monitor.device_tracker] Data device tracker received: {'is connected': True, 'mac': '582D3460DE98', 'rssi': -64, 'gateway_id': 'ble_monitor'}
2022-05-11 21:29:20 INFO (Thread-4) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 582d3460de98: 043e350d0113000098de60342d580100ff7fb40000000000000000001b0201061716cdfd881298de60342d580201640f01890904d3000000
2022-05-11 21:29:20 DEBUG (MainThread) [custom_components.ble_monitor.binary_sensor] Data binary sensor received: {'rssi': -76, 'packet': 137, 'battery': 100, 'illuminance': 211, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}
2022-05-11 21:29:20 DEBUG (MainThread) [custom_components.ble_monitor.sensor] Data measuring sensor received: {'rssi': -76, 'packet': 137, 'battery': 100, 'illuminance': 211, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}
2022-05-11 21:29:20 DEBUG (MainThread) [custom_components.ble_monitor.device_tracker] Data device tracker received: {'is connected': True, 'mac': '582D3460DE98', 'rssi': -76, 'gateway_id': 'ble_monitor'}
2022-05-11 21:29:22 INFO (Thread-4) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 582d3460de98: 043e350d0113000098de60342d580100ff7fb40000000000000000001b0201061716cdfd881298de60342d580201640f018b0904d3000000
2022-05-11 21:29:22 DEBUG (MainThread) [custom_components.ble_monitor.binary_sensor] Data binary sensor received: {'rssi': -76, 'packet': 139, 'battery': 100, 'illuminance': 211, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}
2022-05-11 21:29:22 DEBUG (MainThread) [custom_components.ble_monitor.sensor] Data measuring sensor received: {'rssi': -76, 'packet': 139, 'battery': 100, 'illuminance': 211, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}
2022-05-11 21:29:22 DEBUG (MainThread) [custom_components.ble_monitor.device_tracker] Data device tracker received: {'is connected': True, 'mac': '582D3460DE98', 'rssi': -76, 'gateway_id': 'ble_monitor'}
```

**Comment by Ernst79** [comment]
The first isn't parsed, the second one does, but only contains illuminance and battery. 

`result  ({'rssi': -76, 'packet': 137, 'battery': 100, 'illuminance': 211, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}, None)`

I found the issue. will fix it in the next release. After the fix, the first message is being parsed as 

`result  ({'rssi': -64, 'packet': 134, 'motion': 0, 'illuminance': 209, 'mac': '582D3460DE98', 'type': 'CGPR1', 'firmware': 'Qingping', 'data': True}, None)`

**Comment by Ernst79** [comment]
Should be fixed in 8.6.5

**Comment by baggein** [comment]
Thank you, BLE BATTERY QINGPR1 now shows the correct condition of the battery. The previous version was 100% all the time.
